src/homework5/task1.py

```diff
@@ -0,0 +1,27 @@
+""" Оформите решение задач из прошлых домашних работ в функции.
+    Напишите функцию runner.
+    a.	runner() – все фукнции вызываются по очереди
+    b.	runner(‘func_name’) – вызывается только функцию func_name.
+    c.	runner(‘func’, ‘func1’...) - вызывает все переданные функции
+"""
+
+
+import func_task1
+
+
+def runner(*args):
+    if args:
+        for func_name in args:
+            getattr(func_task1, func_name)()
+
+    else:
```

Comment:
it works with only 3 functions with appropriate name. Solution should be holistic